students/randi_peterson/session03/slicing_lab.py

```diff
@@ -0,0 +1,69 @@
+def exchange_firstandlast(seq):
+#Return sequence with first and last elements exchanged
+    seq_first = seq[0:1]
+    seq_last = seq[-1:]
+    seq_middle = seq[1:-1]
+    new_seq = seq_last + seq_middle + seq_first
+    return new_seq
+
+def remove_everyother(seq):
+    #Return sequence with every other item removed
+    new_seq = seq[0:len(seq):2]
+    return new_seq
```

Comment:
can simplify to `return seq[::2]`

you can omit the default values of slicing.